.github/workflows/kind.yml

```diff
@@ -71,7 +71,17 @@ jobs:
         sudo mv kind /usr/local/bin
     - name: Run e2e tests
       run: |
-        ./ci/kind/test-e2e-kind.sh --encap-mode encap
+        mkdir log
+        ANTREA_LOG=$PWD/log ./ci/kind/test-e2e-kind.sh --encap-mode encap
```

Comment:
could we rename the variable to `ANTREA_LOGS_DIR`?

Reply:
Done.